### PR TITLE
Update non-windows.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about  support for AWS and GCP-native compute

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11189

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/non-windows.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ Organizations face threats across a variety of platforms and devices. Our teams
 
 With Microsoft Defender for Endpoint, customers benefit from a unified view of all threats and alerts in the Microsoft 365 Defender portal, across Windows and non-Windows platforms, enabling them to get a full picture of what's happening in their environment, which empowers them to more quickly assess and respond to threats.
 
+> [!NOTE]
+> Microsoft Defender for Endpoint does not support native compute workloads in Amazon Web services(AWS) and Google Cloud Platform(GCP).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/non-windows.md]:
```suggestion
> Microsoft Defender for Endpoint does not support native compute workloads in Amazon Web Services (AWS) and Google Cloud Platform (GCP).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**